.github/workflows/ci.yml

```diff
@@ -155,10 +155,10 @@ jobs:
           save-if: ${{ github.event_name != 'merge_group' }}
 
       - name: cargo clippy
-        run: cargo hack clippy --workspace --locked --target ${{ matrix.target }} --optional-deps --each-feature --ignore-unknown-features --features std -- -D warnings
+        run: cargo hack clippy --package parley --package fontique --locked --target ${{ matrix.target }} --optional-deps --each-feature --ignore-unknown-features --features std -- -D warnings
```

Comment:
Hmm. Why do we want to not run clippy on the `xtask`  code itself? Is it just how much longer it takes (since it needs to check axum, etc.), or something else?

Reply:
I am disabling it only for andoid, as kompari does not compile there because of:

```
     Compiling libdeflate-sys v1.23.1
  warning: libdeflate-sys@1.23.1: Compiler family detection failed due to error: ToolNotFound: failed to find tool "x86_64-linux-android-clang": No such file or directory (os error 2)
  error: failed to run custom build command for `libdeflate-sys v1.23.1`
```

Reply:
Ah - that's unfortunate, but not too important at the moment.